plot-api/src/commonMain/kotlin/jetbrains/letsPlot/FrontendContext.kt

```diff
@@ -8,4 +8,6 @@ package jetbrains.letsPlot
 interface FrontendContext {
     fun display(plotSpecRaw: MutableMap<String, Any>)
     fun getInfo() = "Not available"
-}
\ No newline at end of file
+}
+
+expect fun createDefaultFrontendContext(): FrontendContext
```

Comment:
Lets have this declaration in a separate source file.

Reply:
Done